tests/benchmarks/power_consumption/adc/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
missing header?

Reply:
if license header was meant, I've added it for all CmakeLists files.